samples/excel/89-preview-apis/style.yaml

```diff
@@ -0,0 +1,241 @@
+order: 10
+id: excel-style
+name: Style
+description: 'Add, apply, get and delete styles.'
+author: siewmoi
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-new-style").click(() => tryCatch(addNewStyle));
+        $("#apply-new-style").click(() => tryCatch(applyNewStyle));
+        $("#apply-built-in-style").click(() => tryCatch(applyBuiltInStyle));
+        $("#get-style-font").click(() => tryCatch(getStyleFontProperties));
+        $("#get-style-alignment").click(() => tryCatch(getStyleAlignmentProperties));
+        $("#delete-new-style").click(() => tryCatch(deleteNewStyle));
+
+        async function addNewStyle() {
+            await Excel.run(async (context) => {
+                let styles = context.workbook.styles;
+
+                // Add a new style to the style collection.
+                // Styles is in the Home tab ribbon.
+                styles.add("Diagonal Orientation Style");
+                styles.getItem("Diagonal Orientation Style").orientation = 38;
```

Comment:
Why do you set the orientation and addIndent up here instead of down where you set the other 3 properties? If there's a good reason, please use a code comment to explain. I find that the function works fine if I comment out these 2 lines and add the following 2 lines down where the other properties are set:
        newStyle.orientation = 38;
        newStyle.addIndent = true;